src/ui/server/helpers/remove-commas-from-string/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * removes commas from string
+ * @param {string} string
+ * @returns {string} with commas replaced
+ */
+const removeCommasFromString = (string: string) => string.replace(/,/g, '');
+
+export default removeCommasFromString;
```

Comment:
Was going to suggest we move this to `helpers/string` - turns out we already have a function for this! `stripCommas`